pkgs/development/python-modules/pyezminc/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildPythonPackage, isPy3k, fetchFromGitHub, stdenv,
+  netcdf, hdf5, libminc, ezminc,
+  cython, numpy, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "pyezminc";
+  version = "1.2";
+  name = "${pname}-${version}";
+ 
+  disabled = isPy3k;  # ??
+  # sort of works but the example programs fail due to unicode issues, etc.
```

Comment:
Did you try setting these:
`LC_ALL = "en_US.utf-8";`
`nativeBuildInputs = [ glibcLocales ];`